itou/www/itou_staff_views/views.py

```diff
@@ -77,20 +90,112 @@ def export_job_applications_unknown_to_ft(
         def content():
             yield job_app_export_spec.keys()
             for job_app in job_apps_qs:
-                yield job_app_export_row(job_app)
+                yield export_row(job_app_export_spec, job_app)
 
         # Avoid exceedingly long filenames.
         departments_str = "multiple_departements" if len(departments) >= 5 else "_".join(departments)
-        export_ts = f"{timezone.localdate().strftime('%Y-%m-%d')}_{timezone.localtime().strftime('%H-%M-%S')}"
         writer = csv.writer(Echo())
         return StreamingHttpResponse(
             content_type="text/csv",
             headers={
                 "Content-Disposition": content_disposition_header(
                     as_attachment=True,
-                    filename=f"candidats_emplois_inclusion_{departments_str}_non_certifies_{export_ts}.csv",
+                    filename=f"candidats_emplois_inclusion_{departments_str}_non_certifies_{get_export_ts()}.csv",
                 ),
             },
             streaming_content=(writer.writerow(row) for row in content()),
         )
     return render(request, template_name, {"form": form})
+
+
+@login_required
+def export_ft_api_rejections(request):
+    if not request.user.is_superuser:
+        raise Http404
+
+    first_day_of_month = timezone.now().replace(day=1, hour=0, minute=0, second=0, microsecond=0)
+    rejected_approvals = (
+        Approval.objects.filter(
+            pe_notification_status="notification_error",
+            pe_notification_time__range=[
+                first_day_of_month - relativedelta(months=1),
+                first_day_of_month,
+            ],
+        )
+        .select_related("user", "user__jobseeker_profile")
+        .order_by("pe_notification_time")
+    )
+
+    if len(rejected_approvals) == 0:
+        messages.add_message(request, messages.WARNING, "Pas de rejets de PASS IAE sur le dernier mois")
```

Comment:
Peut-être `success`, c’est une bonne chose non ?
nit: Il y a un raccourci.
```suggestion
        messages.success(request, "Pas de rejets de PASS IAE sur le dernier mois.")
```